### DISCUSSION TITLE
Discussion 이라는게 있었군요!!

### DISCUSSION THREAD

**Discussion by Huinno-ParkJinHyun** [learningssik/pull-request#31] [Q&A] [answered]
항상 reference 찾을때 issue만 봤었는데.. 덕분에 시야가 더 넓어졌습니다 감사합니다 🙆🏻‍♂️

**Comment by learningssik** [comment, marked as answer]
답변 감사합니다!

이 글이 도움이 되셨다면 본 레파지토리 우측 상단의 Star를 눌러주세요 ^-^
